### DISCUSSION TITLE
UX Improvement - API Rules - Padlock

### DISCUSSION THREAD

**Discussion by cyrilghali** [pocketbase/pocketbase#1058] [Q&A] [answered]
When I configured access to my collections, I was getting the opposite result to what I expected.

The padlock confused me: I thought it indicated the current state, when in reality it indicated the state I would get if I clicked on it. It always makes me think twice about the current state. I think it's more readable to see the current state so you can make the decision to change it or not rather than thinking about the state I'm currently in if the padlock shows my next click will open it. 

As a user, I'll spend more time reading the status than changing it.

This is just my two cents! 

Thanks

**Comment by ganigeorgiev** [comment, marked as answer]
In my opinion it is more clear as it is.

It is a button, not just an icon, and represents the action associated to it (eg. the same as if it was text "Lock"/"Unlock"). In addition, there should be a tooltip text on mouse hover that explains what the button will do. The disabled input and the hint text serves as an indication what is the current state of the input.
